### PR TITLE
data: shubhsh19

### PR DESCRIPTION
**Pull request by shubhsh19**
Added my data to LinkFree.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6510"></a>

### CODE REVIEW

**Comment by priyankarpal** [comment]
cool

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Review by accodes21 — APPROVED** [review]

File: data/shubhsh19.json
Comment (SaraJaoude) [inline data/shubhsh19.json:4]: needs a comma:
```suggestion
  "bio": "Frontend + Devops",
```

**Comment by SaraJaoude** [comment]
@priyankarpal and @accodes21 thank you for reviewing this PR, however you approved it but the JSON did not pass.  Please can you check before giving your approval, to reduce confusion for the author and maintainers.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by shubhsh19** [comment]
Thanks a lot @SaraJaoude

**Comment by priyankarpal** [comment]
> @priyankarpal and @accodes21 thank you for reviewing this PR, however, you approved it but the JSON did not pass. Please can you check before giving your approval, to reduce confusion for the author and maintainers?

sorry for that, I understand. We had a community discussion where @shubhsh19 talked about open-source contributions & created this PR here to teach new users. So I just approved it.

**Pull request merged**
